scripts/ddp-RxTerms-script/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as XLSX from "xlsx";
+import { RxNormDrugOption } from "./models";
+import { parseToHOCON, writeFile } from "./utils/common";
+
+const inputFileName = process.argv[2] || "RxTerms202202.txt";
```

Comment:
I recommend checking that the data file actually exists. So something like this:
```typescript

import * as fs from 'fs';

if (!fs.existsSync(inputFileName)) {
    console.error(`Input file with name ${inputFileName} does not exist`);
    process.exit(1);
}